aten/src/ATen/TensorIndexing.h

```diff
@@ -640,7 +640,7 @@ static inline Tensor get_item(
       tensorIndices,
       disable_slice_optimization,
       self_device,
-      *self_sizes);
+      self_sizes);
```

Comment:
This is a leftover from [#78934](https://github.com/pytorch/pytorch/pull/78934): the size argument has been made optional so no need to dereference anymore